A40-csds-support.md

```diff
@@ -0,0 +1,328 @@
+xDS Configuration Dump via Client Status Discovery Service in gRPC
+----
+* Author(s): lidizheng
+* Approver: markdroth
+* Status: In-Review
+* Implemented in: All languages
```

Comment:
This isn't true yet.  We should add languages here as they actually get implemented.

Reply:
Updated to `Core/C++/Java/Golang (1st release)`